### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by bretterer**

### CODE REVIEW

**Comment by nbarbettini** [comment]
Looks like this readme stuff was also mixed in with another commit (`factor.go`)?

File: README.md [hunk 1]
```diff
@@ -1,93 +1,417 @@
-# Okta Management SDK for Golang
-[![Build Status](https://img.shields.io/travis/okta/okta-sdk-golang/master.svg)](https://travis-ci.org/okta/okta-sdk-golang)
+[<img src="https://devforum.okta.com/uploads/oktadev/original/1X/bf54a16b5fda189e4ad2706fb57cbb7a1e5b8deb.png" align="right" width="256px"/>](https://devforum.okta.com/)
+[![Build Status](https://img.shields.io/travis/okta/okta-sdk-golang.svg?logo=travis)](https://travis-ci.org/okta/okta-sdk-golang)
+[![GitHub](https://img.shields.io/github/license/okta/okta-sdk-github.svg)](https://github.com/okta/okta-sdk-golang)
```
Comment (nbarbettini) [inline README.md]: Remove `/license/` 🙂

File: README.md [hunk 2]
```diff
@@ -1,93 +1,417 @@
...
+
+## Release status
+
+This library uses semantic versioning and follows Okta's [library version policy](https://developer.okta.com/code/library-versions/).
+
+:warning: Beta alert! This library is in beta. Breaking changes may be introduced at minor versions in the 0.x range. Please lock your dependency to a specific version until this library reaches 1.x.
+
+| Version | Status                    |
+| ------- | ------------------------- |
+| 0.x  | :heavy_check_mark: Beta Release               |
```
Comment (nbarbettini) [inline README.md]: Let's remove `heavy_check_mark` (reserved for stable)

**Comment by bretterer** [comment]
@nbarbettini it was mixed with a template fix needed for the readme to work

File: README.md [hunk 3]
```diff
@@ -1,93 +1,417 @@
...
+## Need help?
+
+If you run into problems using the SDK, you can
+
+* Ask questions on the [Okta Developer Forums][devforum]
+* Post [issues][github-issues] here on GitHub (for code errors)
+
+## Getting started
+
+Follow these steps to install the Okta Golang SDK in your project:
```
Comment (nbarbettini) [inline README.md]: Will it always be just this one step? In that case, we could simplify to:

---

To install the Okta Golang SDK in your project, run

```
go get github.com/okta/okta-sdk-golang
```

File: README.md [hunk 4]
```diff
@@ -1,93 +1,417 @@
...
+### Get a User
+```
+config := okta.NewConfig()
+client := okta.NewClient(config, nil, nil)
+user, resp, err := client.User.GetUser(user.Id, nil)
+```
+
+### List all Users
+```
+config := okta.NewConfig()
```
Comment (nbarbettini) [inline README.md]: No need to repeat client instantiation in each example - want to move it above `### Authenticate a user`? I should update my template to reflect this.

Comment (bretterer) [reply README.md]: Personally, I like that it has this in every example... makes it more copy/pasteable?

File: README.md (hunk 1 above)
Comment (bretterer) [reply README.md]: thoughts on why?

Comment (bretterer) [reply README.md]: nevermind... realize it is doubled there

**Pull request merged**